Dockerfile

```diff
@@ -14,15 +14,11 @@ COPY --from=builder /home/target/jpo-aws-depositor-jar-with-dependencies.jar /ho
 
 CMD java -Dlogback.configurationFile=/home/logback.xml \
 	-jar /home/jpo-aws-depositor-jar-with-dependencies.jar \
-	--bootstrap-server $DOCKER_HOST_IP:9092 \
```

Comment:
Are none of these values required to run the app in deployment? The Dockerfile will be used by anyone wanting to deploy the s3 depositor.

Reply:
The values removed include the following:
- K_AWS_ACCESS_KEY_ID
- K_AWS_SECRET_ACCESS_SECRET
- K_AWS_SESSION_TOKEN
- K_AWS_EXPIRATION
- API_ENDPOINT
- HEADER_ACCEPT

These were not necessary for local testing, but may be necessary in other situations. I can add them back into the dockerfile so that they can utilized by developers if necessary.

The API_ENDPOINT option was giving me some trouble (preventing local testing), so we may want to leave that one out or look into why we get an error when attempting to get AWS credentials.

Reply:
Actually the following are optional:
- K_AWS_ACCESS_KEY_ID
- K_AWS_SECRET_ACCESS_SECRET
- K_AWS_SESSION_TOKEN
- K_AWS_EXPIRATION
- HEADER_ACCEPT

Should we add these back into the dockerfile? @payneBrandon 

The API_ENDPOINT does not appear to be optional, but I was only able to get the project to work when not passing it in.

Reply:
These have been added back into the Dockerfile:
- K_AWS_ACCESS_KEY_ID
- K_AWS_SECRET_ACCESS_SECRET
- K_AWS_SESSION_TOKEN
- K_AWS_EXPIRATION
- HEADER_ACCEPT

API_ENDPOINT is still left out, since the program crashes whenever it is passed.